addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/payloader/ExtensionPayloader.java

```diff
@@ -82,6 +92,8 @@ public boolean canUnload() {
     public void unload() {
```

Comment:
It can be fixed in a different PR but it seems we're not unloading `errorCategory`